subworkflows/local/emptydrops_removal.nf

```diff
@@ -0,0 +1,33 @@
+include { CELLBENDER_REMOVEBACKGROUND } from '../../modules/nf-core/cellbender/removebackground'
+include { ADATA_BARCODES              } from '../../modules/local/adata_barcodes'
+
+workflow EMPTY_DROPLET_REMOVAL {
```

Comment:
In this workflow, we could have the user choose between cellbender and emptydrops

Reply:
you mean to still keep the old script we had?

Reply:
I had thought, yes, but we can discuss this again if there's still need for it. 
Does gCBDS still use it or did they switch to cellbender entirey?
Another option would be to remove it altogether and just leave it to scdownstream.

Reply:
for both questions, think the best is that we have a talk with them, regarding how to they feel regarding it
:)